st2actions/st2actions/cmd/actionrunner.py

```diff
@@ -19,9 +20,22 @@
     time=True)
 
 
+def _setup_sigterm_handler():
```

Comment:
Further there is also https://github.com/StackStorm/st2/blob/master/st2common/st2common/signal_handlers.py

I would like to avoid any major refactoring as part of this PR or create another interface/utility. It would be nice to try to collect some of these into a cohesive module and see what approach makes most sense. It can easily become a rabbit hole.